utilities/test_suite/HIP/runAudioTests.py

```diff
@@ -73,7 +73,7 @@ def generate_performance_reports(RESULTS_DIR):
     print(dfPrint_noIndices)
 
 def run_unit_test_cmd(srcPath, case, numRuns, testType, batchSize, outFilePath):
-    print("./Tensor_audio_hip " + srcPath + " " + str(case) + " " + str(numRuns) + " " + str(testType) + " " + str(numRuns) + " " + str(batchSize))
+    print("\n./Tensor_audio_hip " + srcPath + " " + str(case) + " " + str(numRuns) + " " + str(testType) + " " + str(numRuns) + " " + str(batchSize))
```

Comment:
Why this is added?
if there is a requirement
can't we move this outside this print statement?

Reply:
This was added for an empty line before printing the contents. Without it, the output does not look good.